### ISSUE TITLE
Add datetimepicker to simple_form initializer

### ISSUE THREAD

**Issue by maebeale** [maebeale/mutual-aid#171]

**Comment by leahriffell** [comment]
Hi @maebeale! Could you please assign this issue to me? Do you have any additional context for us or any wireframes to be used?
I see a datetime picker under /shifts/new but not sure where else this is utilized.

**Comment by maebeale** [comment]
Thanks! We wanted a SimpleForm "component" so when we call an input,  we can say "as: :datetime" in views.

**Comment by maebeale** [comment]
(Is that helpful, @leahriffell )

**Comment by leahriffell** [comment]
Yes, thank you @maebeale! We'll start digging in and let you know if we have further questions. Thank you!
